### DISCUSSION TITLE
How long to verify domain ownership?

### DISCUSSION THREAD

**Discussion by mark-wiemer** [microsoft/vsmarketplace#995] [Q&A]
Hi, I submitted a request to verify markwiemer.com weeks ago and haven't received any updates. Might it have been lost somewhere?

https://marketplace.visualstudio.com/manage/publishers/mark-wiemer

![image](https://github.com/user-attachments/assets/83786b4e-0b1b-4345-aeea-62ea2ae0e2d3)

Thanks,

Mark

**Comment by mark-wiemer** [comment]
I see https://code.visualstudio.com/api/working-with-extensions/publishing-extension#verify-a-publisher has some docs:

> Once your TXT record has been validated, the Marketplace team will review your request and let you know the result within 5 business days. The validation includes, but is not limited to: domain, website and extensions content eligibility, legitimacy, trust and positive reputation. New publishers will need a consistent track record demonstrating these for at least 6 months.

I feel I meet that, but I never heard back :(

**Reply by mark-wiemer** [reply]
After emailing support, they say my site isn't related to my extensions, so it isn't eligible. I've asked for clarity.

**Reply by Aluisio** [reply]
@mark-wiemer I also had a lot of trouble verifying my domain. I spoke to a Microsoft member here https://github.com/microsoft/vscode-discussions/discussions/1090 and after several emails with support and adjustments, my domain was verified (and I didn't even get a response that it had been verified, the verification simply appeared). However, on 09/10/24, the verification simply disappeared. I redid the DNS configuration and contacted them again via email and so far I have not had any response.

**Reply by mark-wiemer** [reply]
Yeah, seems this process is not well-optimized. I plan to update my site to reference my extension directly, then email back, but I still haven't heard back since asking for clarity about 2 weeks ago :(

**Reply by Aluisio** [reply]
Coincidentally after I replied here, I received an update email. But unfortunately the text of the email was the standard text that the request was for review and that it is to wait

**Reply by TFWol** [reply]
@mark-wiemer Don't you work for Microsoft? Weird that you'd be getting the runaround.

**Reply by mark-wiemer** [reply]
Haha I used my personal email, gotta keep those things separate. Never got an update, btw. Still planning on updating my site and re-submitting, but that may be a few months out

**Reply by TFWol** [reply]
> Never got an update, btw. 

Wow, that's garbage.
Maybe if you include the word "AI" in the title somewhere? 😆

**Reply by danilopezmella** [reply]
Same issue here. It's been a month for me with no news. Im thinking about a way to redirect my domain to the public repo of my extension and hope for the best.

**Reply by mark-wiemer** [reply]
FYI, I have finally updated my site to include a ref to AHK++ (markwiemer.com/ahkpp), I'll keep this thread posted when I re-submit for verified domain ownership, but it's not super high on my to-do list right now

**Reply by mark-wiemer** [reply]
For anyone stumbling upon this, I've updated https://markwiemer.com/ahkpp and re-sent an email. The Marketplace UX was a bit janky and did not let me re-submit or save any changes despite multiple attempts. Crossing fingers, but this still isn't the biggest fish I'm frying :D

**Reply by mark-wiemer** [reply]
Excerpt of their email response:

> We have validated your TXT record. However, we noticed that your website is not related to your extensions, it currently contains more personal details.Also one of your extensions is not updated from 2 years.  Please update the website according to the [VS Marketplace policy](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#verify-a-publisher) and resubmit for verification.

My reply:

> Can you please quote the section of that doc that disallows personal information? And the section that mentions extensions being out of date? I have read it multiple times now. Nowhere does it mention personal info or extensions needing to be updated.

The closest excerpt from the doc I can find:

> The validation includes, but is not limited to: domain, website and extensions prerequisites for track record, content eligibility, legitimacy, trust and positive reputation.

So basically if VS doesn't like your site they can deem it invalid and force the "untrusted" banner on all extension installs, even though they know it's you because they validated your TXT record.
